website/pages/docs/deployment/ecs.md

```diff
@@ -78,189 +62,227 @@ Once you have a CloudQuery API Key you are going to create a Secret in AWS Secre
+      Path: "/"
+      Policies:
+      - PolicyName: SchedulerExecutionRole
+        PolicyDocument:  
+          Version: '2012-10-17'
+          Statement:
+          - Action:
+            - ecs:RunTask
+            Effect: Allow
+            Resource: !Ref ScheduledWorkerTask
+            Condition:
+              ArnEquals:
+                ecs:cluster: !GetAtt ECSCluster.Arn
+          - Effect: Allow
+            Action:
+              - iam:PassRole
+            Resource: !GetAtt ExecutionRole.Arn
+
+  ####### IAM role for Fargate execution#####
+  ExecutionRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: ecs-tasks.amazonaws.com
+            Action: 'sts:AssumeRole'
+      Policies:
+        - PolicyName: AccessAPIKeySecret
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - secretsmanager:GetSecretValue
+                Resource: !Sub ${CQApiKey}
+        - PolicyName: WriteDataToS3Destination
+          PolicyDocument: 
+            Version: 2012-10-17
+            Statement:
+              - Effect: Allow
+                Action:
+                  - s3:PutObject
+                Resource: !Sub arn:${AWS::Partition}:s3:::${DestinationS3Bucket}/*
+        - PolicyName: DenyData
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Effect: Deny
+                NotResource: 
+                  - !Sub arn:${AWS::Partition}:s3:::${DestinationS3Bucket}/*
+                Action:
+                  - s3:GetObject
+              -
+                Effect: Deny
+                Resource: '*'
+                Action:
+                  - cloudformation:GetTemplate
+                  - dynamodb:GetItem
+                  - dynamodb:BatchGetItem
+                  - dynamodb:Query
+                  - dynamodb:Scan
+                  - ec2:GetConsoleOutput
+                  - ec2:GetConsoleScreenshot
+                  - kinesis:Get*
+                  - lambda:GetFunction
+                  - logs:GetLogEvents
+                  - sdb:Select*
+                  - sqs:ReceiveMessage
+      ManagedPolicyArns:
+        - 'arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy'
+        - 'arn:aws:iam::aws:policy/ReadOnlyAccess'
+        - 'arn:aws:iam::aws:policy/AWSMarketplaceMeteringFullAccess'
+Outputs:
+  ClusterId:
+    Value: !Ref ECSCluster
+  TaskArn:
+    Value: !Ref ScheduledWorkerTask
 ```
 
-Replace the `REPLACE_WITH_S3_DESTINATION_BUCKET` placeholder with the name of the S3 bucket you want to use to store the data. 
-
 
-This policy will allow the ECS task to write data to the S3 bucket you specified while also ensuring that CloudQuery never has access to any of your data.
+## Step 3: Deploy the CloudFormation Template
```

Comment:
Step 3 is duplicated here

Reply:
Fixed in [caeb720](https://github.com/cloudquery/cloudquery/pull/18964/commits/caeb720af516f894e04dd7b5746a7ded81b81bed)